src/pkg/provisioner/provisioner.go

```diff
@@ -17,24 +17,69 @@
 package provisioner
 
 import (
+	"context"
 	"fmt"
+	"log"
 	"os"
+	"os/exec"
+
+	"cloud.google.com/go/storage"
+	"golang.org/x/sys/unix"
 )
 
+func setup(systemd *systemdClient) error {
+	log.Println("Setting up environment...")
+	if err := systemd.stop("update-engine.service"); err != nil {
+		return err
+	}
+	if err := unix.Mount("", "/root", "tmpfs", 0, ""); err != nil {
+		return fmt.Errorf("error mounting tmpfs at /root: %v", err)
+	}
+	cmd := exec.Command("docker-credential-gcr", "configure-docker")
```

Comment:
should `docker-credential-gcr` also be part of the `Deps` structure?

Reply:
Done.

Reply:
Missed one spot in the rebase...this isn't done yet

Reply:
Now this is done.